### ISSUE TITLE
Radio stream sounds slow (also, speed change when changing audio output)

### ISSUE THREAD

**Issue by giquo** [Taiko2k/Tauon#896]
System: Fedora 37 6.0.12-602.inttf.fc37.x86_64

- I have this radio station stream URL: [https://play10.tikast.com:20113/stream](url) and when I start playing, it plays at slow speed, therefore the audio sounds deeper.
Didn't found anything in settings to make it play at "normal speed".

NOW

- I find out something else, I was using my headphones connected a Focusrite 4i4 output while listening music, when I changed to the normal PC speakers, the Audio went from normal speed to slower.
- For testing purposes, I closed the program, opened again while using the PC audio output (speakers) and then changed the output to the Focusrite, the audio went faster.

The hint I got in my mind is something to do with the bitrate or some biz

**Comment by Taiko2k** [comment]
Yep I can replicate the issue. I have committed a fix now that will be in upcoming v7.4.7.

**Comment by C0rn3j** [comment]
Closing as fixed.
